autoreject/autoreject.py

```diff
@@ -84,7 +98,11 @@ def validation_curve(epochs, y, param_name, param_range, cv=None):
                          param_name="thresh", param_range=param_range,
                          cv=cv, n_jobs=1, verbose=0)
```

Comment:
Since we're at it, perhaps let's expose the b_jobs param.

Reply:
what's that? :)

Reply:
probably the `n_jobs` parameter for the `validation_curve` function: Currently hardcoded to 1, see line 85(old)/99(new)